fre/pp/configure_script_yaml.py

```diff
@@ -30,165 +31,94 @@ def yamlInfo(yamlfile,experiment,platform,target):
-                  f.write(f"grid=regrid-xy/{gridLat}_{gridLon}\n")
-                if "static" in compvalue:
-                  f.write("freq=P0Y\n") 
-              elif compkey == "sources":
-                f.write(f"{compkey}={compvalue} ")
-              elif compkey == "timeSeries":
-                for i in compvalue:
-                  for key,value in i.items():
-                    if key == "source":
-                      f.write(f"{value} ")
-
-            # Create/write regrid rose app
-            with open(regrid_roseapp,'a') as f:
-              if i.get("xyInterp") != None: 
-                if compkey == "type":
-                  f.write(f"\n[{compvalue}]\n")
-                  if "atmos" in compvalue:
-                    f.write("inputRealm=atmos\n")
-                  elif "land" in compvalue:
-                    f.write("inputRealm=land\n")
-                  elif "river" in compvalue:
-                    f.write("inputRealm=land\n")
-                  elif "ocean" in compvalue:
-                    f.write("inputRealm=ocean\n")
-                  elif "aerosol" or "tracer" in compvalue:
-                    f.write("inputRealm=atmos\n")
-                elif compkey == "sourceGrid": 
-                  f.write(f"inputGrid={compvalue}\n")
-                elif compkey == "interpMethod":
-                  f.write(f"{compkey}={compvalue}\n")
-                elif compkey == "sources":
-                  f.write(f"{compkey}={compvalue} ")
-                  try: 
-                    i["timeSeries"]
-                    for elem in i['timeSeries']:
-                      for key,value in elem.items():
-                        if key == "source":
-                          f.write(f"{value} ")
-                  except:
-                    print("No timeseries information")
-
-                  f.write("\n")
-
-                if compkey == "xyInterp" and compvalue == y["defaultxyInterp"]:
-                  f.write(f"outputGridType=default\n")
-                elif compkey == "xyInterp": 
-                  gridLat=compvalue.split(",")[0]
-                  gridLon=compvalue.split(",")[1]
-
-                  f.write(f"outputGridLat={gridLat}\n")
-                  f.write(f"outputGridLon={gridLon}\n")
-                  f.write(f"outputGridType={gridLat}_{gridLon}\n") 
+      for i in value:
+        comp = i.get('type')
+        sources = i.get('sources')
+        interp_method = i.get('interpMethod')
+
+        # set remap items
+        rose_remap.set(keys=[f'{comp}', 'sources'], value=f'{sources}')
+        # if xyInterp doesnt exist, grid is native
+        if i.get("xyInterp") == None:
+          rose_remap.set(keys=[f'{comp}', 'grid'], value='native')
+        # if xyInterp exists, component can be regridded
+        else:
+          interp_split = i.get('xyInterp').split(',')
+          rose_remap.set(keys=[f'{comp}', 'grid'], value=f'regrid-xy/{interp_split[0]}_{interp_split[1]}.{interp_method}')
```

Comment:
Just checking for clarification and for if this is what we wanted. For grid, the output would look like this:
```grid=regrid-xy/180_288.conserve_order2```

Instead of previously, like this, for example:
```grid=regrid-xy/180_288```

Reply:
Yes, I think so. the remap files are particular to the interpolation method